pages/blog/index.vue

```diff
@@ -0,0 +1,113 @@
+<template>
+  <b-container data-qa="blog">
+    <b-row class="flex-md-row pb-5">
+      <b-col cols="12">
+        <h1>{{ $t('blog.blogs') }}</h1>
+      </b-col>
+      <b-col cols="12">
+        <b-card-group
+          class="card-deck-4-cols"
+          deck
+          data-qa="blog posts"
+        >
+          <ContentCard
+            v-for="(post, index) in posts"
+            :key="index"
+            :title="post.fields.headline"
+            :url="{ name: 'blog-all', params: { pathMatch: post.fields.identifier } }"
+            image-url="https://picsum.photos/600/300/?image=25"
+            :description="post.fields.description"
```

Comment:
The description doesn't exist (yet?) in the blog content model. Should this be created?

Reply:
After speaking with @rwd it was agreed to put this in place

Reply:
There is a description now, the first card/blog post on the foyer page has one in Contentful. The ContentCard does not accept a "description", but "texts" which is an array.

Reply:
Could you use the texts and get rid of the description in the ContentCard? The texts array was meant for displaying different texts (eg a description). You can see an example in the BrowseContentCard component.